styles.js

```diff
@@ -8,39 +19,42 @@ export default createGlobalStyle`
   }
 
 html {
---globalAppBackgroundColor: #272440;
---globalPlantBackgroundColor: #30400D;
---globalPlaceBackgroundColor: #591202;
---globalDateBackgroundColor: #BF3605;
+--globalAppBackgroundColor: #1C3D59;
+--globalPlantBackgroundColor: #59551B;
+--globalPlaceBackgroundColor: #8C470B;
+--globalDateBackgroundColor: #8C1616;
 --globalLightColor: #FFDAB9;
 --globalSecondLightColor: #D3D2D2;
 --globalShadowColor: #39355a;
---globalNavigationBackgroundColor: #3C4973;
+--globalNavigationBackgroundColor: ##BF871F;
 --globalBorderShadow: var(--globalShadowColor) 0px 1px 0px,
     rgba(255, 255, 255, 0.25) 0px 1px 0px inset;
+--scriptfond: ${indie.style.fontFamily}
 }
 
 body {
     margin: 0;
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: ${jost.style.fontFamily};
+
+    /* font-family: Arial, Helvetica, sans-serif; */
```

Comment:
You can remove this too :)

Reply:
Thanks for the tip. I usually do that before I create a PR. I'm still working on the components, so such code snippets stay a bit in their place.

Reply:
✅

Reply:
Sorry, I have only now realized that the Ändeurngen are still in the PR arrived. I had planned to publish this later. Thanks for the review.